.gitignore

```diff
@@ -4,3 +4,4 @@ diskmaker
 !diskmaker/
 cover.out
 _output
+.vscode/
```

Comment:
nit: strictly speaking this change can be made to your global `$HOME/.gitignore` file rather than in this file.

Reply:
removed